README.md

```diff
@@ -1,6 +1,95 @@
-# MoneyWise2020
+# Money Wise
```

Comment:
Perhaps make it easier for staff to see what it stands for. i.e. **Team Name:** Money Wise